wrappers/python/nntile/nntile_core/tensor.pyi

```diff
@@ -0,0 +1,55 @@
+from collections.abc import Buffer
+from typing import Sequence
+
+from .tile import Tile
+
+class TensorTraits:
+    """Integer arithmetics for tensors."""
+
+    def __init__(self, shape: Sequence[int], basetile_shape: Sequence[int]):
+        ...
+
+    def __repr__(self) -> str: ...
+
+    @property
+    def basetile_shape(self) -> tuple[int, ...]: ...
+
+    @property
+    def grid(self) -> Tile: ...
```

Comment:
Actual type of the returned value is "TileTraits"

Reply:
Seems like this Python include file was not run by a Python. How can we test such files?

Reply:
The purpose of these files is actually testing, lol. Stub files define types then type checker like `mypy` or `pylint` verifies that type system is consistent.

Reply:
If purpose of such files is testing then why there was no testing? Running the code would show the error.